### PR TITLE
[SMALLFIX] Fix an integration test with -Phadoop-1

### PR DESCRIPTION
**Pull request by ifcharming**

### CODE REVIEW

**Comment by ifcharming** [comment]
@aaudiber @peisun1115

File: tests/src/test/java/alluxio/hadoop/FileSystemAclIntegrationTest.java [hunk 1]
```diff
@@ -137,8 +137,15 @@ public void chmod() throws Exception {
...
     Assert.assertTrue(sUfs.exists(PathUtils.concatPath(sUfsRoot, fileA)));
-    // Default permission should be 0644
-    Assert.assertEquals((short) 0644, fs.getPermission().toShort());
+
+    if (sUfs instanceof HdfsUnderFileSystem && HadoopClientTestUtils.isHadoop1x()) {
+      // If the UFS is hadoop 1.0, the org.apache.hadoop.fs.FileSystem.create uses default
+      // permission option 0777.
+      Assert.assertEquals((short) 0777, fs.getPermission().toShort());
+    } else {
+      // Default permission should be 0644
```
Comment (aaudiber) [inline tests/src/test/java/alluxio/hadoop/FileSystemAclIntegrationTest.java]: nit: end with period

**Comment by peisun1115** [comment]
LGTM

File: tests/src/test/java/alluxio/hadoop/FileSystemAclIntegrationTest.java (hunk 1 above)
Comment (ifcharming) [inline tests/src/test/java/alluxio/hadoop/FileSystemAclIntegrationTest.java]: done, thanks

**Comment by aaudiber** [comment]
Jenkins, test this please

**Pull request merged**